### PR TITLE
Bump dependencies, use Rollup v2

### LINKED ISSUE #122: Warning about `generateBundle`
Hi. I’m seeing this warning whenever I build my site with rollup:

```
src/main.js → public/build/bundle.js...
(!) A plugin is directly adding properties to the bundle object in the "generateBundle" hook. This is deprecated and will be removed in a future Rollup version, please use "this.emitFile" instead.
created public/build/bundle.js in 7.6s
```

I started with `sveltejs/template`, and then added Smelte, and that’s when I began to see this. If I comment out the Smelte stuff, the warning goes away.

### LINKED ISSUE #179: [Sapper] "chunk.imports is not iterable" on fresh install
Can't use smelte on a fresh sapper installation, throws `chunk.imports is not iterable`

```sh
npx degit "sveltejs/sapper-template#rollup" app
cd app
node scripts/setupTypeScript.js
npm i
npm i -S smelte
```
[rollup.config.js](https://gist.github.com/karlos1337/7f6a7b2e34253a08ef2d6b6e9c44d9a0)

```sh
npm run dev
```
imagen

### PR DESCRIPTION
**Pull request by tcd93**
fixes #179
fixes #122

### CODE REVIEW

**Comment by tcd93** [comment]
Netlify uses an old Node version, my machine built without error (Node v14)

**Comment by tcd93** [comment]
Also, I think we should add some sort of warnings when user did not have `tailwindcss` installed as dependency (in client-side-only build, or with [svelte-template](https://github.com/sveltejs/template))

**Comment by i8ramin** [comment]
@matyunya is this project still actively being developed?

**Comment by matyunya** [comment]
I'm planning to upgrade to Tailwind 2 once it's compatible with the latest PostCSS. For my personal projects I'm using https://github.com/matyunya/headlong. I was trying to solve component customization problem for a while, but ended up using headlong (no pain with tw setup and such), and simply copypasting Smelte components as a boilerplate for my current needs.

I sure do accept PRs and do occasional bugfixes but I still don't see how Smelte v2 should look like.

**Comment by matyunya** [comment]
Thanks for your pull request though!

File: rollup.config.js [hunk 1]
```diff
@@ -122,15 +134,18 @@ export default {
       string({
         include: "**/*.txt"
       }),
-      resolve(),
-      alias({ smelte: "src/index.js" }),
+      resolve({
+				dedupe: ['svelte']
```
Comment (benmccann) [inline rollup.config.js:138]: looks like there's a mix of tabs and spaces here

**Comment by huyz** [comment]
> I'm planning to upgrade to Tailwind 2 once it's compatible with the latest PostCSS.

@matyunya  Doesn't Tailwind 2 support PostCSS version 8? https://tailwindcss.com/docs/upgrading-to-v2#install-tailwind-css-v2-0-and-post-css-8

**Pull request merged**